pkg/codegen/internal/test/testdata/output-funcs/go/mypkg/funcWithListParam.go

```diff
@@ -42,14 +40,14 @@ func FuncWithListParamOutput(ctx *pulumi.Context, args FuncWithListParamOutputAr
 
 type FuncWithListParamOutputArgs struct {
 	A pulumi.StringArrayInput `pulumi:"a"`
-	B pulumi.StringPtrInput `pulumi:"b"`
+	B pulumi.StringPtrInput   `pulumi:"b"`
```

Comment:
Do we know why the whitespace changed?

Reply:
I think something is applying gofmt now?

A large benefit of this change is that these tests now use official codegen API. Before that, the tests hooked into undocumented internals to build out half-baked support for testing function codegen. Perhaps official API does a gofmt pass.